scripts/scripts.js

```diff
@@ -217,4 +220,14 @@ async function loadPage() {
   loadDelayed();
 }
 
+export function decorateInnerHrefButtonsWithArrowIcon(block) {
+  block.querySelectorAll('a').forEach((a) => {
+    const icon = domEl('calcite-icon', { icon: 'arrowRight', scale: 's' });
+    icon.style.marginInlineStart = '4px';
+    if (a.href.includes('esri.com')) {
```

Comment:
An inner href can be to "esri.com" or "main--esri..." (in .page and .live), can you make sure to take that into account?